File: keiner_lindarte_aro/instructor_henry/mis_programas/dart/orientado_a_objetos/2_clase_de_31_julio/actividad.dart
class Videojuego {

  String? genero;
  String? plataformas;
  String? empresa;

  Videojuego (String empresa){
    this.empresa = empresa;
  }

}

void main () {

  Videojuego zelda = new Videojuego("nintendo");
  zelda.genero = "aventura";
  zelda.plataformas = "playstation, xbox y pc";

  print("la empresa ${zelda.empresa} creo un juego con el genero ${zelda.genero} para las plataformas ${zelda.plataformas}");


}

File: keiner_lindarte_aro/instructor_henry/mis_programas/ruby/orientado_a_objetos/2_clase_de_31_julio/actividad.rb
class Videojuego 

  attr_accessor :genero, :plataformas, :empresa

  def initialize(empresa)
    self.empresa = empresa
  end

end

zelda = Videojuego.new ("nintendo")
zelda.genero = "aventura"
zelda.plataformas = "playstation, xbox y pc"

puts "La empresa #{zelda.empresa} creo un juego con el genero #{zelda.genero} para las plataformas #{zelda.plataformas}"
